README.md

```diff
@@ -8,14 +8,29 @@ Expected environment:
 - (WIP) Onload/master
 - (WIP) OCP 4.12 with:
   - Node Feature Discovery (NFD) Operator
-  - Kernel Module Management (KMM) Operator
+  - Kernel Module Management (KMM) Operator ([Redhat docs](https://docs.openshift.com/container-platform/4.12/hardware_enablement/kmm-kernel-module-management.html), [codebase docs](https://kmm.sigs.k8s.io/documentation/deploy_kmod/))
 
-### SFC out of tree driver
+### SFC out-of-tree driver
 
-The recommended way of installing onload's sfc driver is using [KMM](https://github.com/rh-ecosystem-edge/kernel-module-management/)
+Installing the `sfc` out-of-tree driver is optional. Depending on the Linux Kernel version included in the underlying distribution (eg. RHCOS) and use case demands, the in-tree kernel `sfc` driver may be sufficient. This section can be skipped without affecting the other deployment steps.
+
+If more recent driver features are desired, follow [Deploy using KMM](#deploy-using-kmm) to dynamically load an out-of-tree `sfc` driver before accelerated workloads are started, and additionally follow [Day-0 MachineConfig](#day-0-deployment-using-machineconfig) if driver features are required at boot time.
```

Comment:
The out of tree driver is **essential** for onload. It is only day 1/0 loading that is optional.